### PR TITLE
Update Repo Mgmt & IAM roadmap sections

### PR DESCRIPTION
**Pull request by ryphil**

### CODE REVIEW

File: content/strategy-goals/strategy/iam/Q2FY23-OKR.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Q2 Objective and Key Results
+
+### Problem Area: Align on Team Vision
+
+Today, the IAM team is less than two weeks old and does not yet have a long-term mission or vision that aligns with the company vision.
```
Comment (unknwon) [inline content/strategy-goals/strategy/iam/Q2FY23-OKR.md]: >the IAM team is less than two weeks old

Not so true? 🤣

File: content/strategy-goals/strategy/iam/index.md [hunk 2]
```diff
@@ -1,53 +1,22 @@
...
+This page outlines the vision, strategy, and goals of the [Identity and Access Management team](../../../departments/engineering/teams/iam/index.md).
 
-## Q2 Objective and Key Results
+## FY 2023
 
-### Problem Area: Align on Team Vision
+Note: Sourcegraph's fiscal year runs from February 1, 2022 to January 31, 2023.
 
-Today, the IAM team is less than two weeks old and does not yet have a long-term mission or vision that aligns with the company vision.
+Our main focus for FY23 is on significantly improving the admin experience. We must provide an efficient and secure way for administrators to specify permissions for repositories and our FY23 focus is to significantly improve this experience. Specifically, we are focused on...
```
Comment (unknwon) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
Our main focus for FY23 is on significantly improving the admin experience. We must provide an efficient and secure way for administrators to specify permissions for repositories and our FY23 focus is to significantly improve this experience. Specifically, we are focused on:
```

File: content/strategy-goals/strategy/iam/index.md [hunk 3]
```diff
@@ -1,53 +1,22 @@
...
 
-- Key Result 1: An updated handbook page for the IAM team including mission, vision, and team working agreements.
-- Key Result 2: A GitHub board outlining the key efforts on our 12-month roadmap validated by customer product gaps.
+- Deliver the highest impact improvements to the Admin Experience for repo syncing as agreed upon with CE
+- Deliver new tooling for admins to troubleshoot their instances and invest in training CS to solve more issues on their own
+- Improve the reliability and usability of the Explicit Permissions API and launch support for permissions mechanisms in parallel
 
-**Engineering OKR Alignment:** This does not align to engineering OKRs. It is necessary due to the formation of a new team.
+This also includes a number of non-customer facing improvements...
```
Comment (unknwon) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
This also includes a number of non-customer facing improvements:
```

**Review by unknwon — APPROVED** [review]

File: content/strategy-goals/strategy/iam/Q2FY23-OKR.md (hunk 1 above)
Comment (ryphil) [reply content/strategy-goals/strategy/iam/Q2FY23-OKR.md]: Good catch - Updated!

File: content/strategy-goals/strategy/iam/Q2FY23-OKR.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Within this area, in Q2 we are not working on:
+
+- Implementing RBAC for administrators
+- Updating features to respect our permissions, this will be owned by each feature team
+
+**Engineering OKR Alignment:** Objective #2: Focus on time-to-value for a single use-case (Fixing Vulnerabilities). With a single permissions model, Sourcegraph feature teams will be able to enable cross-feature experiences for our customers to fix vulnerabilities. For example, setting up a code monitor to ensure a vulnerability is not re-introduced after fixing with a batch change.
+
+### Problem Area: Improve Admin Analytics
+
+_Update_: The MVP of this work is planning to be launched in the July release of Sourcegraph (version 3.42).
```
Comment (jplahn) [inline content/strategy-goals/strategy/iam/Q2FY23-OKR.md]: Should we remove "planning to be" from launched?

File: content/strategy-goals/strategy/iam/index.md [hunk 5]
```diff
@@ -1,53 +1,22 @@
...
 
-### Problem Area: Align on Team Vision
+Note: Sourcegraph's fiscal year runs from February 1, 2022 to January 31, 2023.
 
-Today, the IAM team is less than two weeks old and does not yet have a long-term mission or vision that aligns with the company vision.
+Our main focus for FY23 is on significantly improving the admin experience. We must provide an efficient and secure way for administrators to specify permissions for repositories and our FY23 focus is to significantly improve this experience. Specifically, we are focused on:
 
-- Key Result 1: An updated handbook page for the IAM team including mission, vision, and team working agreements.
-- Key Result 2: A GitHub board outlining the key efforts on our 12-month roadmap validated by customer product gaps.
+- Deliver the highest impact improvements to the Admin Experience for repo syncing as agreed upon with CE
```
Comment (jplahn) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
- Deliver the highest impact improvements to the Admin Experience for repo permissions syncing as agreed upon with CE
```

File: content/strategy-goals/strategy/iam/index.md [hunk 6]
```diff
@@ -1,53 +1,22 @@
...
+- Deliver new tooling for admins to troubleshoot their instances and invest in training CS to solve more issues on their own
```
Comment (jplahn) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
- Deliver new tooling for admins to troubleshoot their instances and invest in training CS to solve more issues without engineering support
```

File: content/strategy-goals/strategy/iam/index.md [hunk 7]
```diff
@@ -1,53 +1,22 @@
...
+- Improve the reliability and usability of the Explicit Permissions API and launch support for permissions mechanisms in parallel
 
-**Engineering OKR Alignment:** This does not align to engineering OKRs. It is necessary due to the formation of a new team.
+This also includes a number of non-customer facing improvements:
 
-### Problem Area: Unifying Feature Permissions
+- Fix paper cuts and hacks to improve overall reliability of existing repo syncing and permissions syncing
```
Comment (jplahn) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
- Fix paper cuts and hacks to improve the overall reliability of existing repo permissions syncing functionality 
```

File: content/strategy-goals/strategy/iam/index.md [hunk 8]
```diff
@@ -1,53 +1,22 @@
...
+- Dramatically improve our automated integration and regression testing for everything we own to gain great confidence in the quality of what we currently own and launch
```
Comment (jplahn) [inline content/strategy-goals/strategy/iam/index.md]:
```suggestion
- Dramatically improve our automated integration and regression testing for everything we own to increase confidence in the quality of what we currently own and launch
```

**Review by jplahn — APPROVED** [review]
I suggested some changes that are relatively minor

File: content/strategy-goals/strategy/iam/Q2FY23-OKR.md (hunk 4 above)
Comment (ryphil) [reply content/strategy-goals/strategy/iam/Q2FY23-OKR.md]: Fixed! Good catch.

**Pull request merged**